frontend/src/metabase/components/form/CustomForm/CustomFormSubmit.tsx

```diff
@@ -4,7 +4,7 @@ import _ from "underscore";
 
 import ActionButton from "metabase/components/ActionButton";
 
-import { FormLegacyContext, LegacyContextTypes } from "./types";
+import { CustomFormLegacyContext, LegacyContextTypes } from "./types";
```

Comment:
FYI, now that prettier is updated we can use `import type`. I'm not sure what _exactly_ it does differently, but it makes it much clearer when you're reading it that certain things are just importing types instead of functioning code.